includes/tables_integrity.json

```diff
@@ -9,23 +9,23 @@
     },
     {
         "table_name": "background_subtasks",
-        "structure_hash": "54f2e975e46967ae42ae5402f68f6748db047a9b7c6cd4beffb2d18155a9acf3"
+        "structure_hash": "7b4eb36d263132dd07e87cb27bc8018dfaeb83a74ac4c155cf1d365c49dceb98"
```

Comment:
No relation with this PR but I wonder if it makes sense: there is the auto increment value in the `SHOW CREATE TABLE` so with each new line, the hash changes systematically.

AUTO_INCREMENT=3050 here:
```
mysql> show create table teampass_nested_tree \G
*************************** 1. row ***************************
       Table: teampass_nested_tree
Create Table: CREATE TABLE `teampass_nested_tree` (
  `id` bigint unsigned NOT NULL AUTO_INCREMENT,
  `parent_id` int NOT NULL,
  `title` varchar(255) NOT NULL,
  `nleft` int NOT NULL DEFAULT '0',
  `nright` int NOT NULL DEFAULT '0',
  `nlevel` int NOT NULL DEFAULT '0',
  `bloquer_creation` tinyint(1) NOT NULL DEFAULT '0',
  `bloquer_modification` tinyint(1) NOT NULL DEFAULT '0',
  `personal_folder` tinyint(1) NOT NULL DEFAULT '0',
  `renewal_period` int NOT NULL DEFAULT '0',
  `fa_icon` varchar(100) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'fas fa-folder',
  `fa_icon_selected` varchar(100) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'fas fa-folder-open',
  `categories` longtext NOT NULL,
  `nb_items_in_folder` int NOT NULL DEFAULT '0',
  `nb_subfolders` int NOT NULL DEFAULT '0',
  `nb_items_in_subfolders` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  KEY `nested_tree_parent_id` (`parent_id`),
  KEY `nested_tree_nleft` (`nleft`),
  KEY `nested_tree_nright` (`nright`),
  KEY `nested_tree_nlevel` (`nlevel`),
  KEY `personal_folder_idx` (`personal_folder`)
) ENGINE=InnoDB AUTO_INCREMENT=3050 DEFAULT CHARSET=utf8
1 row in set (0,00 sec)
```


Also, I haven't seen the use of this anywhere.

Reply:
Correct, I will remove this in the future commits.
I had a plan to use it but not really needed.